src/lib/releaseTestPlanGenerator.spec.ts

```diff
@@ -0,0 +1,80 @@
+import { sortTestCases, TestCase } from "./testPlan";
+import {
+   generateReleaseTestPlan,
+   loadTestPlanFromFile,
+} from "./releaseTestPlanGenerator";
+
+const TEST_DATA_DIR = "./test_data";
+
+describe("releaseTestPlanGenerator", () => {
+   const versionToTest = "0.40.00-v201811141000";
+   const goodTestPlanDir = "./test_data/test_plans";
+   const testCaseOne = {
+      featureName: "My Feature A",
+      isRequired: true,
+      requiredBy: "story-2",
+   };
+   const testCaseTwo = {
+      featureName: "My Feature B",
+      isRequired: true,
+      requiredBy: "story-2",
+   };
+   const testCaseThree = {
+      featureName: "My Feature A",
+      isRequired: false,
+      requiredBy: "story-2",
+   };
+
+   it("sorts test cases", () => {
+      const sortedTestCases: TestCase[] = sortTestCases([
+         testCaseThree,
+         testCaseTwo,
+         testCaseOne,
+      ]);
+
+      expect(sortedTestCases.length).toBe(3);
+      expect(sortedTestCases[0].featureName).toBe(testCaseOne.featureName);
+
+      expect(sortedTestCases[1].featureName).toBe(testCaseThree.featureName);
+
+      expect(sortedTestCases[2].featureName).toBe(testCaseTwo.featureName);
```

Comment:
Minor, `toBe()` should only be used when testing for object identity equality (like when testing for true or false or a number).  Also, I think a more concise way of writing a test for an array like this is: 

```suggestion
      expect(sortedTestCases[2].featureName).toEqual([
        testCaseOne.featureName,
        testCaseTwo.featureName,
        testCaseThree.featureName]);
```

Reply:
These are not equivalent though?  The value of sortedTestCases[2].featureName is not an array of feature names.

Reply:
oh, sorry, that should be `sortedTestCases.map(f => f.featureName)`

Reply:
Oh that's cool!  Done.